src/client/pages/CycleHome/Introduction/Introduction.tsx

```diff
@@ -37,27 +34,9 @@ const Introduction: React.FC = () => {
           <div>{t('landing.about.selectArea')}</div>
 
           <div className="home-area-selector">
-            <div className="home-area-selector__group">
-              <img alt="" src="/img/iconGlobal.svg" />
-              <Link
-                className="home-link m-r"
-                to={Routes.Country.generatePath({ countryIso: Global.WO, assessmentName, cycleName })}
-              >
-                {t(`area.${Global.WO}.listName`)}
-              </Link>
-            </div>
-
-            <div className="home-area-selector__group">
-              <img alt="" src="/img/iconRegions.svg" />
-              <div>{t('common.regions')}</div>
-              <AreaSelector includeRegions={['fra2020', 'secondary']} placeholder="common.select" />
-            </div>
-
-            <div className="home-area-selector__group">
-              <img alt="" src="/img/iconCountries.svg" />
-              <div>{t('common.countries')}</div>
-              <AreaSelector includeCountries placeholder="common.select" />
-            </div>
```

Comment:
Added logic so moved to separate components to keep clean:

Global

Regions

Countries